Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rouille"
-version = "1.0.3"
+version = "2.0.0"
```

Comment:
Not just yet ;) Please revert, otherwise it will automatically publish 2.0.0 when this PR gets merged.

Reply:
Will do. I'll squash this in.